src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -282,25 +282,33 @@ export async function poll(
              */
             const assumedCountryCode = getSignupLocaleCountry(recipient);
 
-            // The unit tests are currently too complex for me to write
-            // a proper test for this, and I need to understand the code
-            // better to be able to refactor it to make it more amenable
-            // to simple tests. Hence, I don't have a test for this yet:
-            /* c8 ignore next 3 */
-            if (typeof recipient.onerep_profile_id === "number") {
-              await refreshStoredScanResults(recipient.onerep_profile_id);
+            const upSell =
+              // double eq checks both null and undefined
+              // eslint-disable-next-line eqeqeq
+              recipient.fxa_uid != null &&
+              isEligibleForPremium(assumedCountryCode) &&
+              !hasPremium(recipient);
+            let dataSummary: DashboardSummary | undefined = undefined;
+            if (upSell) {
+              // The unit tests are currently too complex for me to write
+              // a proper test for this, and I need to understand the code
+              // better to be able to refactor it to make it more amenable
+              // to simple tests. Hence, I don't have a test for this yet:
+              /* c8 ignore next 3 */
```

Comment:
nit(non-blocking): I'd avoid first-person in comments, it's not immediately clear who wrote it.

Reply:
I didn't write this comment, it's just indented